.github/workflows/ota-update.yml

```
@@ -53,6 +53,14 @@ jobs:
         run: |
           eas update --channel ${{ steps.ch.outputs.channel }} --non-interactive --message "Auto OTA: ${GITHUB_SHA::7} (${GITHUB_REF_NAME})"
 
+  notify-skip:
+    needs: detect-native-change
+    if: needs.detect-native-change.outputs.native_changed == 'true'
```

actionlint (1.7.7)

property "native_changed" is not defined in object type {}

(expression)